### ISSUE TITLE
Releases for other architectures? (ARM)

### ISSUE THREAD

**Issue by keithy** [dhall-lang/dhall-lang#970]
Are there likely to be any available for aarch64?

**Comment by sjakobi** [comment]
Do you mean releases of the Haskell implementation? The Haskell implementation has it's own issue tracker: https://github.com/dhall-lang/dhall-haskell/issues

**Comment by Gabriel439** [comment]
Yeah, I'll assume that this was intended for the Haskell implementation and relocate the issue there
